nixos/modules/services/networking/nbd.nix

```diff
@@ -0,0 +1,152 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.nbd;
+  configFormat = pkgs.formats.ini { };
+  iniFields = with types; attrsOf (oneOf [ bool int float str ]);
+  serverConfig = configFormat.generate "nbd-server-config"
+    ({
+      generic = ({
+        user = "root";
+        group = "root";
+        port = cfg.server.listenPort;
+      }
+      // cfg.server.extraOptions
```

Comment:
all `extraOptions` should probably be the first in these update chains, otherwise an extraOption could overwrite a module option with no diagnostics.

Reply:
Done.